runelite-api/src/main/java/net/runelite/api/Client.java

```diff
@@ -223,15 +223,15 @@
 
 	NPCComposition getNpcDefinition(int npcId);
 
-	Area[] getMapAreas();
+	MapIcon[] getMapIconTypes();
```

Comment:
I'm a little worried about changing anything in the `Client.java` as this has to mixin with the runescape client.

Reply:
I've done testing with this on an internal branch, the renaming is necessary to accurately reflect what is being returned.

Reply:
Furthermore all the renamed methods are unused in the current code base